src/widget/woverview.cpp

```diff
@@ -379,6 +387,11 @@ void WOverview::slotLoadingTrack(TrackPointer pNewTrack, TrackPointer pOldTrack)
 void WOverview::onEndOfTrackChange(double v) {
     //qDebug() << "WOverview::onEndOfTrackChange()" << v;
     m_endOfTrack = v > 0.0;
+    // Note that this is needed for making the eot background and frame occur
+    // simultaneoulsy, but this may cause the frame's first blink period being
+    // notably shorter than 1 second since "end_of_track" is not activated in sync
+    // with the 500 ms timer's 'on' period. Maybe use a dedicated timer.
+    m_drawEndOfTrack = v > 0.0;
```

Comment:
A solution would be to make m_drawEndOfTrack an interger in the range of 0..3 advance it every 500 ms and us it instead of v in onEndOfTrackBlinkTimerChange